testutils/factories/oauth2Token.ts

```diff
@@ -10,14 +10,23 @@ interface TokenParams {
 }
 
 class Oauth2TokenFactory extends Factory<string, TokenParams> {
-  deliusToken() {
+  probationPractitionerToken() {
     return this.transient({
       authSource: 'delius',
       userID: '2500128586',
       username: 'joe.smith',
       roles: ['ROLE_PROBATION'],
     })
   }
+
+  serviceProviderToken() {
+    return this.transient({
+      authSource: 'auth',
+      userID: '6c4036b7-e87d-44fb-864f-5a06c1c492f3',
```

Comment:
@rgforsyth do any of the properties of this user matter to you other than `authSource` and `roles`? i.e. do you need a known user ID or username?